qpc/cred/add.py

```diff
@@ -22,6 +11,8 @@
 from qpc.request import POST
 from qpc.translation import _
 
+log = getLogger("qpc")
```

Comment:
I noticed that all of your loggers are "qpc". Is there a reason specifically for them all to be the same? If we use `__name__` instead (see suggested change), then in the future if we want finer control, we could adjust each package or module's log level _independently_ in the project-wide logging config. If they're all using the same logger name like "qpc", though, you have to change all or none.

Also, though this is more of a style choice, I think the prevailing convention is to call the Logger object `logger`. Of course, if you do that, be sure to replace any `log` references with `logger`.

```suggestion
logger = getLogger(__name__)
```

See also: https://docs.python.org/3/howto/logging.html#advanced-logging-tutorial

> A good convention to use when naming loggers is to use a module-level logger, in each module which uses logging, named as follows:
> 
> `logger = logging.getLogger(__name__)`
> 
> This means that logger names track the package/module hierarchy, and it’s intuitively obvious where events are logged just from the logger name.